index.js

```diff
@@ -163,19 +151,18 @@ const resetScores = () => {
 
 function disableStart(status) {
   let startStatus = status;
-  if(startStatus == true) {
-    startButton.disabled = true;
-  }
-  else {
-    startButton.disabled = false;
-  }
 
+  const stBtn = startStatus === true ? true : false;
+  startButton.disabled = stBtn;
+  
   for (let i = 0; i < addPointButton.length; i++) {
-    if(startStatus == true) {
-      addPointButton[i].disabled = false;
-    }
-    else {
-      addPointButton[i].disabled = true;
-    }  
+    const ptBtnStatus = startStatus === true ? false : true;
```

Comment:
- [ ] Here the startButton disabled state and the addPointButton disabled state is depending on the startStatus. You dont have to create a new variable here, instead you can just assign 
`startButton.disabled = startStatus ;` and `addPointButton[i].disabled = !startStatus;`

Reply:
resolved